### PR TITLE
Simplify WNP templates

### LINKED ISSUE #14949: FXDE in-product pages include navbar
When compared to **release/beta**:
https://www-dev.allizom.org/en-CA/firefox/127.0/whatsnew/
https://www-dev.allizom.org/en-CA/firefox/126.0beta/whatsnew/
and **nightly**:
https://www-dev.allizom.org/en-US/firefox/80.0a1/whatsnew/

the WNPs and firstrun for **aurora/developer** currently start with the common layout masthead:
https://www-dev.allizom.org/en-US/firefox/102.0a2/whatsnew/
https://www-dev.allizom.org/en-US/firefox/92.0a2/whatsnew/
https://www-dev.allizom.org/en-US/firefox/83.0a2/firstrun/

(comparison aurora–beta:)
Screenshot 2024-08-01 at 15 07 34
(whatsnew aurora × nightly:)
Screenshot 2024-08-01 at 14 54 17
(firstrun aurora × nightly:)
Screenshot 2024-08-01 at 15 07 34

Is it intentional, or for consistency with other in-product pages this should also skip the masthead?

### PR DESCRIPTION
**Pull request by janbrasna**
## One-line summary

Moves the boilerplate ceremony to just base template, making sure the default metas are always used at least as a fallback.

## Significant changes and points to review

For consistency/simplicity, this also changes:
- all FXDE WNPs and firstrun have navbar (`site_header`) removed now, too;
- adds meta description to nightly, repurposing one of the content headings;
- deletes old font variants from WNP114 that's been removed;
- links current (short) fx logos from `img/logos` where they're now available.

## Issue / Bugzilla link

Resolves #14949
Refs https://github.com/mozilla/bedrock/pull/14720#discussion_r1657022464
Refs https://github.com/mozilla/bedrock/pull/14822#discussion_r1681076314

## Testing

http://localhost:8000/en-US/firefox/131.0/whatsnew/?branch=experiment-wnp-131-tabs&variant=v1
http://localhost:8000/en-CA/firefox/131.0/whatsnew/?branch=experiment-wnp-131-tabs&variant=v2 
http://localhost:8000/en-GB/firefox/131.0/whatsnew/?branch=experiment-wnp-131-tabs&variant=v3 http://localhost:8000/de/firefox/131.0/whatsnew/?branch=experiment-wnp-131-tabs&variant=v4
http://localhost:8000/es-ES/firefox/130.0/whatsnew/
http://localhost:8000/en-CA/firefox/129.0/whatsnew/
http://localhost:8000/en-GB/firefox/129.0/whatsnew/
http://localhost:8000/en-US/firefox/128.0/whatsnew/?branch=experiment-wnp-128-na-pip&variant=v1
http://localhost:8000/en-US/firefox/128.0/whatsnew/
http://localhost:8000/pl/firefox/128.0/whatsnew/?v=1
http://localhost:8000/de/firefox/128.0/whatsnew/?v=2
http://localhost:8000/fr/firefox/128.0/whatsnew/?v=3
http://localhost:8000/en-CA/firefox/127.0/whatsnew/
http://localhost:8000/it/firefox/127.0/whatsnew/
http://localhost:8000/en-US/firefox/126.0/whatsnew/
http://localhost:8000/it/firefox/126.0/whatsnew/
http://localhost:8000/en-CA/firefox/126.0beta/whatsnew/
http://localhost:8000/de/firefox/126.0beta/whatsnew/
http://localhost:8000/en-US/firefox/125.0/whatsnew/
http://localhost:8000/es-ES/firefox/125.0/whatsnew/
http://localhost:8000/firefox/66.6/whatsnew/
http://localhost:8000/en-US/firefox/99.0a1/whatsnew/
http://localhost:8000/en-US/firefox/102.0a2/whatsnew/
http://localhost:8000/en-US/firefox/92.0a2/whatsnew/
http://localhost:8000/en-US/firefox/83.0a2/firstrun/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/developer/firstrun.html [hunk 1]
```diff
@@ -24,6 +24,8 @@
 {% block body_id %}firefox-developer{% endblock %}
 {% block body_class %}firefox-developer firefox-developer-firstrun{% endblock %}
 
+{% block site_header %}{% endblock %}
```
Comment (janbrasna) [inline bedrock/firefox/templates/firefox/developer/firstrun.html:27]: This now makes aurora whatsnew and firstrun aligned with release/beta and nightly in a way that it doesn't show the main site header and navigation (wnp implicitly from the base diff here, firstrun explicitly on this line).

**Review by janbrasna — COMMENTED** [review]
Notes for reviewer:

**Comment by janbrasna** [comment]
Updated to include WNP 130–131 now, too.

**Review by stephaniehobson — APPROVED** [review]
R+ This looks great. Clean and through as always. Nice code cleanup.

@janbrasna In the opening comment you mention needed to add 131 nimbus URLs and further down you say 131 is done. Does that mean the 131 nimbus experiments are done too?

**Comment by janbrasna** [comment]
@stephaniehobson That's a good catch! The template is covered, only at the time of the WIP comment there was a little bit of fiddling with the experiments vs. defaults/fallback etc. but this should be all sorted out now and I've included a bunch of experiment branches across regions in the testing links (which was the only thing missing — I never remember how the nimbus branches go into the query string;D…)

**Pull request merged**